cli/user/src/main/java/io/pravega/cli/user/stream/StreamCommand.java

```diff
@@ -183,7 +181,7 @@ public void execute() throws Exception {
             }
 
             @Cleanup
-            val factory = EventStreamClientFactory.withScope(scopedStream.getScope(), ClientConfig.builder().controllerURI(getControllerUri()).build());
+            val factory = EventStreamClientFactory.withScope(scopedStream.getScope(), getClientConfig());
```

Comment:
nit: The term `factory` is a bit abstract. A `clientFactory` is more precise, and `eventStreamClientFactory` even more so. In general, variable names should be as precise as possible (for reasons like reducing the work that a code reader needs to perform so that the person may focus more on the logic than the syntax). Here, it's not so much an issue as the `factory` is used right under this statement, so you may choose to leave it as-is.

Reply:
done